server/models/db/school.js

```diff
@@ -0,0 +1,26 @@
+const _ = require('lodash');
+
+const { db } = require('../../lib/connection');
+
+const UserTypes = require('../../constants/userTypes.json');
+const Tables = require('../../constants/tables.json');
+
+// This method returns the corresponding workers given school ID. If the school ID is incorrect, it will return null. 
+// If either the tableName or Object are null or undefined, an error will be thrown.
+async function getWorkerIdsForSchool(schoolId, Object) {
+    if (_.isNil(schoolId) ) return null;
+    
+    //Query to recieve worker IDs from Service Worker School Table 
+    const data = await db.any(`select worker_id from Service_Worker_School where school_id='${schoolId}';`);
+
+    //Retrieve worker attributes such as name, phone number, etc 
+    const data = await db.any(`select worker_id from Service_Worker';`);
```

Comment:
Hi Simran.
I would recommend renaming the variable to `workerIds` in line 14 to differentiate it from the data in line 17. Then, I would leverage the `workerIds` in the second query to select from WHERE to ensure that only the relevant worker information are being returned. If possible, I would suggest breaking these two queries into separate methods to better illustrate their different purposes and for higher reusability in the future. To elaborate, the new method I wouls suggest adding would be `getWorkersUsingIds` containing your second query to fetch the workers using an array of workerIds.

Reply:
Hi Amy! Thank you for your time to review this PR! Peter and I have resolved this issue by putting the two queries in separate methods!